helm/defectdojo/templates/celery-beat-deployment.yaml

```diff
@@ -36,6 +36,9 @@ spec:
         envFrom:
         - configMapRef:
             name: {{ $fullName }}
+        - secretRef:
```

Comment:
the extrasecret is injected everywhere except the initializer job. which is the opposite of the "regular" secret which is injected only in the initializer. 

Is it on purpose? what's an example of use of this extraSecret?

Reply:
This can be done, but it will introduce complexity. From  attack surface point of view, the biggest risk is on django frontend, and it is basicly  heart o the system, if it will be compromised everything else is as well.

The purpose of extra secret is to store, secrets that are needed by setting.py e.g. OAUTH or in my case I'm using it to store AES keys for backup, and I want to keep it generic.

Reply:
other pods could be compromised to for various reasons (running on a compromised node, vulnerability in the component like rabbitmq...); however OK to leave it as is for now, there are already variables injected into pods that are not required.